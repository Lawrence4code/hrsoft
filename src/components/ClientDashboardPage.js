import React from 'react';
import { Link } from 'react-router-dom';
import ClientList from './ClientList';
import ClientListFilters from './ClientListFilters';
import ClientSummary from './ClientSummary';

const ClientDashboardPage = () => (
  <div>
    <ClientSummary />
    <ClientListFilters />
    <ClientList />
  </div>
);

export default ClientDashboardPage;
