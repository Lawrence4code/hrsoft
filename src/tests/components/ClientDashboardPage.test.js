import React from 'react';
import { shallow } from 'enzyme';
import ClientDashboardPage from '../../components/ClientDashboardPage';

test('should render ClientDashboardPage', () => {
  const wrapper = shallow(<ClientDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
