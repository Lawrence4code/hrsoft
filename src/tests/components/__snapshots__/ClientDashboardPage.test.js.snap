// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ClientDashboardPage 1`] = `
<div>
  <Connect(ClientListFilters) />
  <Connect(Component) />
</div>
`;
